client/components/domains/register-domain-step/index.jsx

```diff
@@ -381,21 +381,24 @@ class RegisterDomainStep extends React.Component {
 	}
 
 	renderTldButtons() {
+		const isKrackenUi = config.isEnabled( 'domains/kracken-ui/filters' );
```

Comment:
Nitpick, but you could just `return null` if `isKrackenUi` is false, to keep the code clean and reduce nesting below.

Reply:
I considered it, but we already used this pattern in other parts of this file. I'll keep that in mind for our next changes, though.